session/sql_migration_test.go

```diff
@@ -379,3 +387,390 @@ func TestSessionsStoreMigration(t *testing.T) {
 		})
 	}
 }
+
+// randomizedSessions adds 100 randomized sessions to the kvStore, where 25% of
+// them will contain up to 10 linked sessions. The rest of the session will have
+// the rest of the session options randomized.
+func randomizedSessions(t *testing.T, kvStore *BoltStore,
+	accountsStore accounts.Store) {
+
+	ctx := context.Background()
+
+	var (
+		// numberOfSessions is set to 100 to add enough sessions to get
+		// enough variation between randomized sessions, but kept low
+		// enough for the test not take too long to run, as the test
+		// time increases drastically by the number of sessions we
+		// migrate.
+		numberOfSessions = 100
+	)
+
+	for i := 0; i < numberOfSessions; i++ {
```

Comment:
nit: there are a few occurrences here and afterwards where the new loop syntax could be used: `	for i := range numberOfSessions {`

Reply:
Thanks addressed :)!